test-cases/typescript/critical-security-vulnerabilities.ts

```
@@ -0,0 +1,196 @@
+}
+
+function deleteUser(userId: string): boolean {
+    return db.deleteUser(userId);
+}
+
+function processData(data: any, processor: Function): any {
+    return processor(data);
+}
+
+function executeCode(code: string): any {
+    return eval(code);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)